<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddWord</class>
 <widget class="QDialog" name="DialogAddWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>New Word</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>artikel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxArtikel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <property name="currentText">
     <string>---</string>
    </property>
    <item>
     <property name="text">
      <string>---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>der</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>die</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>das</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>word</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWord">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>suffix for plural</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSuffix">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>--</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-n</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-e</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-(n)en</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&quot;-e</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&quot;-er</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEditMeaning">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>meaning</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonAddWord">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>162</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Word</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
